src/main/java/seedu/address/commons/core/GuiSettings.java

```diff
@@ -11,7 +11,7 @@
 public class GuiSettings implements Serializable {
 
     private static final double DEFAULT_HEIGHT = 600;
-    private static final double DEFAULT_WIDTH = 740;
+    private static final double DEFAULT_WIDTH = 1000;
```

Comment:
good width set to account for 3 columns of different lists. However, the default starting list can still be buggy and hide some words if the parameter are too long. For example, some parts of address will be hidden at the start.